prow/README.md

```diff
@@ -155,3 +155,28 @@ $ chmod +x prow/my-presubmit.sh
 This repository (istio/test-infra) also provides Prow jobs.
 
 - `test-infra-presubmit` - run the linting and testing
+
+### Manually Trigger a Prow Job
+
+
+> **Never do this unless necessary AND you are authorized by istio EngProd team (istio.slack.com test-infra channel)**
+
+
+Connect to Prow cluster and then:
```

Comment:
Explain how to connect to a prow cluster

Reply:
This is mainly for our information, not a general education. Plus we don't want random people to connect to prow cluster, should be only us.